App7.2/App7.2/7(2).cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите координаты точки (x, y)");
+
+        var coordinates = Console.ReadLine();
+        var n = coordinates.IndexOf(" ");
+        var x = double.Parse(coordinates.Substring(0, n));
+        var y = double.Parse(coordinates.Substring(n + 1));
```

Comment:
Приглашение к вводу вводит в заблуждение. Из него можно сделать вывод, что координаты нужно вводить в скобках через запятую. А такой ввод вызывает исключение

> Введите координаты точки (x, y)
(0, 1)

C:\...\App7.2\App7.2\bin\Debug\net8.0\App7.2.exe (процесс 27464) завершил работу с кодом -1 (0xffffffff).

Правильно сделать, например, так
```suggestion
        Console.WriteLine("Введите координаты точки через пробел: x y");

        var coordinates = Console.ReadLine();
        var n = coordinates.IndexOf(" ");
        var x = double.Parse(coordinates.Substring(0, n));
        var y = double.Parse(coordinates.Substring(n + 1));
```